### ISSUE TITLE
Couldn't cleanly terminate all actors in 10 seconds!

### ISSUE THREAD

**Issue by dylangrafmyre** [launchdarkly/ruby-client#79]
This gem was added to our Gemfile `ldclient-rb 2.0.3`. Gemspec dependency `ld-celluloid-eventsource` adds `celluliod-io 0.17.3` which depends on `celluloid 0.17.3`.

Running `rails, rake, rspec` commands cause slowness with termination of the task.

```
rake db:migrate
Running via Spring preloader in process 44805
== 20170120043022 EnsureGuestListingsExistForAllInvitations: migrating ========
== 20170120043022 EnsureGuestListingsExistForAllInvitations: migrated (3.1553s)

Annotated (1): app/models/listing_geo_data.rb
E, [2017-01-20T00:32:46.013847 #44805] ERROR -- : Couldn't cleanly terminate all actors in 10 seconds!
```

Please see: https://github.com/celluloid/celluloid/issues/696
This was a regression in `celluloid` and there should be a fix in `0.17.4` and `0.18.0.pre`.

Thanks for looking into possibly updating the `ld-celluloid-eventsource` dependency on `celluloid`.

**Comment by jkodumal** [comment]
Thanks @dylangrafmyre -- we'd been waiting for a final `0.18.0` celluloid release, but we'll give `0.18.0.pre` a try.

**Comment by colmdoyle** [comment]
Any timeline for using https://github.com/launchdarkly/celluloid-eventsource/releases/tag/0.8.4 ?

**Comment by jkodumal** [comment]
@colmdoyle We're testing this out now-- we need to run it through our integration suite and ensure it passes before we release. We'll keep you posted.

**Comment by jkodumal** [comment]
@colmdoyle Things are looking good, and we anticipate that we'll be releasing an updated version tomorrow (Wednesday).

**Comment by jkodumal** [comment]
This should be addressed in v 2.1.0
